spec/integration/extensions/active_record_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Dry::Operation::Extensions::ActiveRecord do
+  include Dry::Monads[:result]
+
+  let!(:model) do
+    Class.new(ActiveRecord::Base) do
+      self.table_name = :foo
+    end
+  end
+
+  before :all do
+    ActiveRecord::Base.establish_connection(adapter: "sqlite3", database: ":memory:")
```

Comment:
Although in memory, do we need an `after :all` removing the table? Thinking about leaving a clean slate for future tests creating the same table.